packages/integrations/mdx/README.md

```diff
@@ -103,6 +103,22 @@ const posts = await Astro.glob('./*.mdx');
 
 See [the official "how MDX works" guide](https://mdxjs.com/docs/using-mdx/#how-mdx-works) for more on MDX variables.
 
+### Exported properties
+
+Alongside your [MDX variable exports](#variables), we generate a few helpful exports as well. These are accessible when importing an MDX file via `import` statements or [`Astro.glob`](https://docs.astro.build/en/reference/api-reference/#astroglob).
+
+#### `file`
+
+The absolute path to the MDX file (e.g. `home/user/projects/.../file.md`).
+
+#### `url`
+
+The browser-ready URL for MDX files under `src/pages/` (e.g. `en/guides/markdown-content/`). For all other MDX files, this will be `undefined`.
```

Comment:
Quick question with re: the `url` example

It looks a little off to see the example here NOT have a forward slash. I get that it's referencing the file in the pages folder, which we do typically write with the trailing slash. But since this isn't a file, I think it's OK to write the URL the way it would be used?

Otherwise, README LGTM!

Reply:
Agree here and what do you think of providing a super explicit example like this?

```suggestion
The browser-ready URL for MDX files under `src/pages/`. For example, `src/pages/en/about.mdx` will provide a `url` of `/en/about/`. For MDX files outside of `src/pages`, `url` will be `undefined`.
```